docs/translatingAddons.md

```diff
@@ -0,0 +1,93 @@
+# Translating NVA Add-ons
+
+This document provides guidance about translating add-ons registered to be translated using the [NVDA's centralized translation system](https://github.com/nvaccess/addon-datastore/blob/master/docs/submitters/submissionGuide.md#RegisteringAnAdd-onInTheTranslationSystem).
+
+Add-ons maintainers can request for their add-ons to be registered on that system, or they may choose other procedures to get their add-ons translated.
```

Comment:
Might be worth clarifying that alternative solutions must be created by the maintainers

```suggestion
Add-ons maintainers can request for their add-ons to be registered on that system, or they may choose their own custom procedure to get their add-ons translated.
```